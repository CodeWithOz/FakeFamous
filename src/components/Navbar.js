import React from 'react';

const Navbar = () => {
  return (
    <nav
      className="ui inverted fluid one item menu"
    ><h1 className="ui item center header">Fake Famous!</h1></nav>
  );
};

export default Navbar;
