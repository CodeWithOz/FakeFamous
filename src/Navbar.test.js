import React from 'react';
import Navbar from './Navbar.js';
import renderer from 'react-test-renderer';

test('Navbar exists', () => {
  const navbar = renderer.create(
    <Navbar />
  );

  // render the navbar
  const tree = navbar.toJSON();
  expect(tree).toMatchSnapshot();
});
