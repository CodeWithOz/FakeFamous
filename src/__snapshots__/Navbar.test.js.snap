// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar exists 1`] = `
<nav
  className="ui inverted fluid one item menu"
>
  <h1
    className="item"
  >
    Fake Famous!
  </h1>
</nav>
`;
